slo_generator/exporters/bigquery.py

```diff
@@ -276,6 +276,11 @@ def _format(errors):
         "type": "FLOAT",
         "mode": "NULLABLE",
     },
+    {
+        "name": "consumed_budget_ratio",
```

Comment:
`error_budget_consumed_ratio` to remain consistent with the naming of similar variables?